### PR TITLE
feat: slideEach-component

### LINKED ISSUE #86: feat: new `SlideEach` global component
### Description

This `SlideEach` compo is a simple wrapper that can wrap each of its child element with `SlideIn` component

## Important Technical Details

this new `SlideEach` global component will take one prop: `children` and some other props which are as same as the props of `SlideIn `component. 
> 1. `SlideIn` is another global component 
> 2. The prop types  of `SlideEach` can be extended from `ISlideIn` `interface` (see SlideIn.tsx to find this `interface`)
> 3. Refer other global component which use `extends` keyword in their props `interface`. You will get a hint about how to write props types for `SlideEach`

the `children` prop of this `SlideEach` compo will have a lots of elements (i.e `children` prop is of type `ReactNode`)

### PR DESCRIPTION
**Pull request by Pratham-Mishra178**
Fixes #86

### CODE REVIEW

**Comment by Pratham-Mishra178** [comment]
Done

File: components/SlideEach/SlideEach.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { SlideIn } from "components/SlideIn";
+import { ISlideIn } from "components/SlideIn/SlideIn";
+import React, { FC, ReactNode } from "react";
+
+interface ISlideEach extends Omit<ISlideIn, "children"> {
```
Comment (ashuvssut) [inline components/SlideEach/SlideEach.tsx:5]: Why did you use Omit here? Omit is unnecessary. There are no type clashes. 
Also, children prop will get inherted from ISlideIn. its also unecessary

correct these 2 unneccasry things

**Review by ashuvssut — CHANGES_REQUESTED** [review]

File: components/SlideEach/SlideEach.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	children: ReactNode;
+}
+
+export const SlideEach: FC<ISlideEach> = ({ children, ...slideProps }) => {
+	return (
+		<>
+			{React.Children.map(children, (child, i) => {
+				return (
+					<SlideIn key={`slide-in=${i}`} {...slideProps}>
```
Comment (ashuvssut) [inline components/SlideEach/SlideEach.tsx:14]: its slide-in- (not slide-in=)

Comment (Pratham-Mishra178) [reply components/SlideEach/SlideEach.tsx:14]: Ok really sorry but it works correctly 😂

Comment (Pratham-Mishra178) [reply components/SlideEach/SlideEach.tsx]: resolved

File: components/SlideEach/SlideEach.tsx (hunk 1 above)
Comment (Pratham-Mishra178) [reply components/SlideEach/SlideEach.tsx]: resolved

File: components/SlideEach/SlideEach.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { SlideIn } from "components/SlideIn";
+import { ISlideIn } from "components/SlideIn/SlideIn";
+import React, { FC, ReactNode } from "react";
+
+interface ISlideEach extends ISlideIn {
+	children: ReactNode;
```
Comment (ashuvssut) [inline components/SlideEach/SlideEach.tsx:6]: @Pratham-Mishra178 
You dont need this `children` prop too! Its already being inherited from `ISlideIn`

Please try to understand what you code does!!!

**Comment by ashuvssut** [comment]
I made that change now. Please understand what you are writing

**Pull request merged**